Detectors/Upgrades/ALICE3/Passive/include/Alice3DetectorsPassive/Pipe.h

```diff
@@ -37,29 +38,33 @@ class Alice3Pipe : public Alice3PassiveBase
   /// Clone this object (used in MT mode only)
   FairModule* CloneModule() const override;
 
-  float getInnerRmin() const { return mBeInnerPipeRmax - mBeInnerPipeThick; }
-  float getInnerRmax() const { return mBeInnerPipeRmax; }
+  float getInnerRmin() const { return mBeInnerPipeRmin; }
+  float getInnerRmax() const { return mBeInnerPipeRmin + mBeInnerPipeThick; }
   float getInnerWidth() const { return mBeInnerPipeThick; }
   float getInnerDz() const { return mInnerIpHLength; }
 
-  float getOuterRmin() const { return mBeOuterPipeRmax - mBeOuterPipeThick; }
-  float getOuterRmax() const { return mBeOuterPipeRmax; }
+  float getOuterRmin() const { return mBeOuterPipeRmin; }
+  float getOuterRmax() const { return mBeOuterPipeRmin + mBeOuterPipeThick; }
   float getOuterWidth() const { return mBeOuterPipeThick; }
   float getOuterDz() const { return mOuterIpHLength; }
 
+  bool getIsAlone() const { return mIsAlone; }
+
  private:
   void createMaterials();
   Alice3Pipe(const Alice3Pipe& orig) = default;
   Alice3Pipe& operator=(const Alice3Pipe&);
 
-  float mBeInnerPipeRmax = 0.;  // inner diameter of the Be section
+  float mBeInnerPipeRmin = 0.;  // inner diameter of the Be section
   float mBeInnerPipeThick = 0.; // inner section  thickness
   float mInnerIpHLength = 0.;   // half length of the inner beampipe around the IP
 
-  float mBeOuterPipeRmax = 0.;  // outer diameter of the Be section
+  float mBeOuterPipeRmin = 0.;  // outer diameter of the Be section
   float mBeOuterPipeThick = 0.; // outer section  thickness
   float mOuterIpHLength = 0.;   // half length of the outer beampipe around the IP
 
+  bool mIsAlone = true; // If A3IP is simulated alone don't subtract TRK shapes
```

Comment:
I think "mIsAlone" is misleading here. The question is only if the inner tracker is present or not, isn't it? We should avoid confusion on what people have to have in e.g. the `build_geometry.C` macro and, there, checking for "TRK" is enough (if I understand correctly).

Reply:
Yes, my very limited view was just considering existing detectors: i.e. TRK. Will change accordingly.